package.json

```
@@ -3,11 +3,13 @@
   "private": true,
   "version": "3.0.0",
   "scripts": {
+    "prepare": "husky",
     "open-browser": "open http://localhost:6100",
     "dev": "next dev -p 6100 & yarn open-browser",
     "build": "yarn run script && next build",
     "start": "next start",
-    "lint": "eslint --ext .tsx --ext .ts .",
+    "lint:prettier": "prettier --write .",
+    "lint:eslint": "eslint --fix",
```

```suggestion
    "lint:prettier": "prettier . --write",
    "lint:eslint": "eslint . --fix",
```
안녕하세요! 지나가다 들렀습니다. 🙋‍♂️

`lint:eslint` script의 경우 `.`가 생략이 된 것 같은데 혹시 의도하신건가요? 
만약 적용이 된다면 `lint:prettier` script도 위와 같이 `.`의 위치를 수정하여 통일하는 것은 어떨까요?

---

아 제가 빠트린것 같아요 ~ 감사합니다. ㅎㅎ